src/cursor/cur_hs.c

```diff
@@ -11,7 +11,7 @@
 
 static int __curhs_prev_visible(WT_SESSION_IMPL *, WT_CURSOR_HS *);
 static int __curhs_next_visible(WT_SESSION_IMPL *, WT_CURSOR_HS *);
-
+static int __hs_cursor_search_near_helper(WT_SESSION_IMPL *, WT_CURSOR *, bool);
```

Comment:
What's the reason behind this naming convention compared with the `__curhs` functions? It seems like an inconsistency.

Reply:
We have renamed a number of functions in cur_hs.c file to make them following the correct naming conventions.